SofaImGui/src/SofaImGui/initSofaImGui.cpp

```diff
@@ -19,7 +19,7 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#include <SofaImGui/config.h>
+#include <SofaImGui/init.h>
```

Comment:
I don't see this file in your PR. Did you forget to push it?

Reply:
yep 🙊
thanks !